host-bin/startxiwi

```diff
@@ -0,0 +1,54 @@
+#!/bin/sh -e
+# Copyright (c) 2015 The crouton Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+APPLICATION="${0##*/}"
+OPTS_ENTER=''
+OPTS_XIWI=''
+
+USAGE="$APPLICATION [options] chroot_app [parameters]
+
+Wraps enter-chroot to launch a window or tab in Chromium OS for any graphical application.
+Applications launched in this way show in independent windows or tabs.
+
+By default, it will use the primary user on the first xiwi-enabled chroot found and launch
+the chroot_app in a window.
+
+Options:
+$(enter-chroot -h 2>&1 | grep -e ' -[bckntu]')
```

Comment:
enter-chroot needs to be found, much like we do down below.

In fact, since you're already making an enterchroot variable, you can just make it sooner and reuse it!  Bonus!